songs/song3_dirty.txt

```diff
@@ -0,0 +1,31 @@
+define noteA is pause(HT) | note(C4,    E)
+define noteB is pause(HT) | note(C4;  E4, E)
+define noteCE is pause(HT) | note(C4;  E4;  G4, E)
+define noteCH is pause(HT)   | note(C4;  E4  ;G4, E)
+define melodyA is pause(HT)   | note(E4, Q) | note(G4, Q) | note(A4, H)
+define melodyB is pause(HT) | note(A4, Q) | note(B4, Q) | note(E4, H)
+define halfMelA is note(D4, E) | note(C4, E)
+define halfMelB is note(D4, E)  || | note(E4, H)
```

Comment:
many | is a syntax error.

Reply:
Sure, but parser is prepared even for this...

Reply:
However if it is critical should exit with syntax error message...